src/iidm/BusBreakerVoltageLevelViews.cpp

```diff
@@ -30,6 +30,10 @@ stdcxx::CReference<Bus> BusBreakerViewImpl::getBus(const std::string& busId) con
 }
 
 stdcxx::Reference<Bus> BusBreakerViewImpl::getBus(const std::string& busId) {
+    const auto& bus = m_voltageLevel.getNetwork().getIndex().find<Bus>(busId);
```

Comment:
I think you can solve your issue using `voltageLevel.getNetwork().find<Bus>(busId)`. This method is in the public API and will delegate the call to the `NetworkIndex`. You should not need to make `BusBreakerViewImpl` and `Network` friend anoymore.

Have you test it's working well for configuredBus (i.e. the bus is found in the index and not through the voltage level objects)

Reply:
Oh thanks 👍 

Yes I checked it works also for ConfiguredBus by debugging it : 
![image](https://user-images.githubusercontent.com/60604989/128490823-bbc7f5b7-d6dd-47e8-80ab-ee799ab562f1.png)